dev/VSIX/ProjectTemplates/Desktop/CSharp/UnitTestApp/UnitTestApp.xaml.cs

```diff
@@ -20,38 +20,37 @@
 // To learn more about WinUI, the WinUI project structure,
 // and more about our project templates, see: http://aka.ms/winui-project-info.
 
-namespace $safeprojectname$
+namespace $safeprojectname$;
+
+/// <summary>
+/// Provides application-specific behavior to supplement the default Application class.
+/// </summary>
+public partial class UnitTestApp : Application
 {
     /// <summary>
-    /// Provides application-specific behavior to supplement the default Application class.
+    /// Initializes the singleton application object.  This is the first line of authored code
+    /// executed, and as such is the logical equivalent of main() or WinMain().
+    /// </summary>
+    public UnitTestApp()
+    {
+        this.InitializeComponent();
+    }
+
+    /// <summary>
+    /// Invoked when the application is launched.
     /// </summary>
-    public partial class UnitTestApp : Application
+    /// <param name="args">Details about the launch request and process.</param>
+    protected override void OnLaunched(Microsoft.UI.Xaml.LaunchActivatedEventArgs args)
     {
-        /// <summary>
-        /// Initializes the singleton application object.  This is the first line of authored code
-        /// executed, and as such is the logical equivalent of main() or WinMain().
-        /// </summary>
-        public UnitTestApp()
-        {
-            this.InitializeComponent();
-        }
-
-        /// <summary>
-        /// Invoked when the application is launched.
-        /// </summary>
-        /// <param name="args">Details about the launch request and process.</param>
-        protected override void OnLaunched(Microsoft.UI.Xaml.LaunchActivatedEventArgs args)
-        {
-            Microsoft.VisualStudio.TestPlatform.TestExecutor.UnitTestClient.CreateDefaultUI();
-
-            m_window = new UnitTestAppWindow();
-            m_window.Activate();
-
-            UITestMethodAttribute.DispatcherQueue = m_window.DispatcherQueue;
-
-            Microsoft.VisualStudio.TestPlatform.TestExecutor.UnitTestClient.Run(Environment.CommandLine);
-        }
-
-        private Window? m_window;
+        Microsoft.VisualStudio.TestPlatform.TestExecutor.UnitTestClient.CreateDefaultUI();
+
+        m_window = new UnitTestAppWindow();
+        m_window.Activate();
+
+        UITestMethodAttribute.DispatcherQueue = m_window.DispatcherQueue;
+
+        Microsoft.VisualStudio.TestPlatform.TestExecutor.UnitTestClient.Run(Environment.CommandLine);
     }
+
+    private Window? m_window;
```

Comment:
```suggestion
    private Window? _window;
```
And, this should go at the top.

Reply:
`m_` changed to `_` and the `private Window? _window;` moved to the top